# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libraw1394/multi-line-string.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libraw1394-0.9.0/src/testlibraw.c.orig-rock-maint	2003-08-30 23:28:39.000000000 +0200
+++ libraw1394-0.9.0/src/testlibraw.c	2003-08-30 23:29:38.000000000 +0200
@@ -19,14 +19,14 @@
 
 #define TESTADDR (CSR_REGISTER_BASE + CSR_CYCLE_TIME)
 
-const char not_compatible[] = "\
-This libraw1394 does not work with your version of Linux. You need a different
-version that matches your kernel (see kernel help text for the raw1394 option to
-find out which is the correct version).\n";
-
-const char not_loaded[] = "\
-This probably means that you don't have raw1394 support in the kernel or that
-you haven't loaded the raw1394 module.\n";
+const char not_compatible[] = 
+"This libraw1394 does not work with your version of Linux. You need a different"
+"version that matches your kernel (see kernel help text for the raw1394 option to"
+"find out which is the correct version).\n";
+
+const char not_loaded[] = 
+"This probably means that you don't have raw1394 support in the kernel or that"
+"you haven't loaded the raw1394 module.\n";
 
 quadlet_t buffer;
 
